wrapper.js

```diff
@@ -225,6 +230,61 @@ exports.Wallet = class Wallet {
         return lib.rgblib_get_address(this.wallet);
     }
 
+    getBtcBalance(online) {
+        const params = { online };
+        const expectedTypes = {
+            online: "object",
+        };
+        validateTypes(params, expectedTypes);
+        return JSON.parse(lib.rgblib_get_btc_balance(this.wallet, online));
+    }
+
+    getAssetBalance(assetId) {
+        const params = { assetId };
+        const expectedTypes = {
+            assetId: "string",
+        };
+        validateTypes(params, expectedTypes);
+        return JSON.parse(lib.rgblib_get_asset_balance(this.wallet, assetId));
+    }
+
+    listBtcTransactions(online) {
```

Comment:
```suggestion
    listTransactions(online) {
```
Please let's keep the naming consistent with rgb-lib Rust methods